css/subject-viewer.styl

```diff
@@ -14,7 +14,7 @@
     width: 100%
 
   .subject-video-controls
-    display: inline-block
+    display: initial
```

Comment:
Maybe scope this change for the subject viewer on the collections pages? We may want `display: inline-block` for the controls where the subject viewer is used elsewhere like the subject talk page or the classifier.